tsfile/src/main/java/org/apache/iotdb/tsfile/read/TsFileSequenceReader.java

```diff
@@ -195,8 +196,8 @@ public String readTailMagic() throws IOException {
    * whether the file is a complete TsFile: only if the head magic and tail magic string exists.
    */
   public boolean isComplete() throws IOException {
-    return tsFileInput.size() >= TSFileConfig.MAGIC_STRING.length() * 2 && readTailMagic()
-        .equals(readHeadMagic());
+    return tsFileInput.size() >= TSFileConfig.MAGIC_STRING.length() * 2 + TSFileConfig.VERSION_NUMBER.length() &&
+        (readTailMagic() + TSFileConfig.VERSION_NUMBER).equals(readHeadMagic());
```

Comment:
In my opinion, I think we can use two methods: `readHeadMagic()` which return "TsFile" and `readVersionNumber()`. We do not need to do operations like "String + String" (e.g., `readTailMagic() + TSFileConfig.VERSION_NUMBER`), which waste memory..